### PR TITLE
Add description field to checklist

### PR DESCRIPTION
**Pull request by jbarronpayit**
Hoping to add a section to the top of the checklist so people are prompted to provide at least a sentence about what they did or are hoping to accomplish.  This should also hopefully prevent every autopopulated slack preview from just being the checklist.

- [x] I have verified that this is not a new project. If it is a new project (new service, new application, new repository), I have contacted the InfoSec team and completed their processes to gain proper approval from the following InfoSec team members:

- [x] I have verified that the backout plan for this change conforms to our standard engineering backout plan located [here](https://payitdev.atlassian.net/wiki/spaces/SEC/pages/2833416205/Standard+Change+Control+Back+Out+Plan). If it does not, I have documented an alternative backout plan below:

- [x] I have verified that this change is backwards compatible. If it is not, I have specified the breaking changes and how they will be handled below:

- [x] I have verified that this change will not impact the security controls built into the application or introduce any new security vulnerabilities. If it will, I have defined the security impact below:

- [x] I have verified that this change will not result in downtime. If it will, I have noted the impact below:

- [x] I have verified that no new dependencies were introduced. If they were, I have vetted them below:

- [x] I have verified that all applicable tests were updated to ensure complete test coverage of any new or modified code.

- [x] I have verified that any relevant documentation such as the README is still up to date and not impacted by my changes. If documentation needs updating for accuracy, I have done so.

### CODE REVIEW

**Review by menzenski — APPROVED** [review]
s/breif/brief

Might also call out linking the Jira ticket?

File: PULL_REQUEST_TEMPLATE.MD [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[Replace this line with a breif description of the work done or expected outcome]
```
Comment (w25r) [inline PULL_REQUEST_TEMPLATE.MD:1]: Spell check... :-)

**Review by w25r — COMMENTED** [review]
I think we should go more bold and do something like
#### Description
## REPLACE WITH BRIEF DESCRIPTION
Also please add the JIRA, if applicable, to the title

And I also think we should get feedback from dev all before forcing this.

File: PULL_REQUEST_TEMPLATE.MD (hunk 1 above)
Comment (MaxBuckley) [reply PULL_REQUEST_TEMPLATE.MD:1]: I would make a "Description" header and then put a placeholder below describing the expectations

Comment (jbarronpayit) [reply PULL_REQUEST_TEMPLATE.MD:1]: Sadly it seems like visual studio code doesn't flag on that and I missed the transposition.  I'll get it fixed.
image

**Comment by jbarronpayit** [comment]
> I think we should go more bold and do something like
> #### Description
> ## REPLACE WITH BRIEF DESCRIPTION
> 
> Also please add the JIRA, if applicable, to the title
> 
> And I also think we should get feedback from dev all before forcing this.

I was 100% going to get feedback from dev all before merging but wanted to get some input from people that were in the recent meeting where this was discussed first.  I'm unsure if having the header of "Description" is also going to get tiring to see in the slack auto-expands but I could add that in too.  I was hoping that leaving the first line as a "Replace this line" sort of thing would get useful information into the slack expand right away.

File: PULL_REQUEST_TEMPLATE.MD [hunk 2]
```diff
@@ -1,3 +1,7 @@
+####Description
```
Comment (jklnr) [inline PULL_REQUEST_TEMPLATE.MD:1]: ####doesn't get rendered as a heading in github markdown anymore. 
```suggestion
#### Description
```

**Comment by jklnr** [comment]
> I think we should go more bold 

Hah @w25r i had basically the opposite feedback. IMO `#### Description` is by itself a very clear suggestion.

Small nit I guess but it would save us the triple click+backspace by omitting the hint text in the first place.

File: PULL_REQUEST_TEMPLATE.MD [hunk 3]
```diff
@@ -1,3 +1,7 @@
+#### Description
+_[Replace this line with a brief description of the work done or expected outcome
+ Also please ensure any applicable jiras are in the title of the PR]_
+
```
Comment (jklnr) [inline PULL_REQUEST_TEMPLATE.MD:4]:
```suggestion

#### Compliance Checklist

```

**Review by chris-clarke-dev — APPROVED** [review]

**Comment by jbarronpayit** [comment]
> > I think we should go more bold
> 
> Hah @w25r i had basically the opposite feedback. IMO `#### Description` is by itself a very clear suggestion.
> 
> Small nit I guess but it would save us the triple click+backspace by omitting the hint text in the first place.

I think I'm currently leaning more towards Will's opinion/ideas on the header + dummy text.  I'm unsure how much worse it would be than going over all of the checklist items and verifying/checking them for the PR.  If it does end up being onerous I'm up for people to further tweak this in the future.  Probably just want to get a PR in front of people for a few days at least like this one so we have a chance for people to leave comments/tweaks on it.

**Pull request merged**